hack/upgrade-test-harness/conf.yaml

```diff
@@ -42,5 +42,5 @@ testParams:
     operatorVersion: 2.12.1
     stackVersion: 8.13.2
   - name: upcoming
-    operatorVersion: 2.13.0-SNAPSHOT
-    stackVersion: 8.14.0-SNAPSHOT
+    operatorVersion: 2.13.0
+    stackVersion: 8.14.0
```

Comment:
I don't think you want to bump this, as you won't be able to run the upgrade test harness, as 8.14 hasn't been released yet...

Reply:
aah thanks!! fixed it.